### DISCUSSION TITLE
Preset Fields and/or collections

### DISCUSSION THREAD

**Discussion by Michael-Nussbaumer** [directus/directus#3410] [Feature Requests]
Lets say you start a new project and have 10 collections to configure. Some of the collections having the same fields. (For example: title, description, image)

It would be nice if you have the possibility to create preset fields or even collections (or make it possible to duplicate a collection?) So when you create a new collection, you can choose which preset fields you want to include.

**Comment by benhaynes** [comment]
I like this! Saving fields presets would be nice, though another option would be to use two optional drop-downs in the first pane of the field creator to select an existing collection/field elsewhere in the system to use... instead of going through the manual setup process.

I can draft a mock-up of this if it gets traction... but seems like it solves the (duplicate/preset) field with one new feature. Not sure if this would work with relational fields (yet) though.
I can

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
